packages/python-google-compute-engine/google_compute_engine/metadata_watcher.py

```diff
@@ -154,33 +154,31 @@ def _GetMetadataUpdate(
 
   def _HandleMetadataUpdate(
       self, metadata_key='', recursive=True, wait=True, timeout=None,
-      retry=True):
+      retries=5):
     """Wait for a successful metadata response.
 
     Args:
       metadata_key: string, the metadata key to watch for changes.
       recursive: bool, True if we should recursively watch for metadata changes.
       wait: bool, True if we should wait for a metadata change.
       timeout: int, timeout in seconds for returning metadata output.
-      retry: bool, True if we should retry on failure.
+      retries: int, Number of times to retry obtaining metadata.
 
     Returns:
       json, the deserialized contents of the metadata server.
     """
     exception = None
-    while True:
+    while retries > -1:
```

Comment:
`>=0` i think is easier to understand here

Reply:
Friendly ping - can you please make this update?